src/styles/createPalette.js

```diff
@@ -77,21 +77,48 @@ export const dark = {
 
 export default function createPalette(palette: Object) {
   const {
-    primary = indigo,
-    secondary = pink,
-    error = red,
+    primary = {
+      light: indigo[300],
+      main: indigo[500],
+      dark: indigo[700],
+    },
+    secondary = {
+      light: pink.A200,
+      main: pink.A400,
+      dark: pink.A700,
+    },
+    error = {
+      main: red[500],
+    },
     type = 'light',
     // Same value used by material-components-web
     // https://github.com/material-components/material-components-web/blob/ac46b8863c4dab9fc22c4c662dc6bd1b65dd652f/packages/mdc-theme/_functions.scss#L49
-    contrastThreshold = 3.1,
+    contrastThreshold = 3,
```

Comment:
3.1 so we better follow the specification. Compare https://material-ui-next.com/style/color/#color with https://material.io/guidelines/style/color.html#color-color-palette.

Reply:
"It's déjà vu all over again". 😄 

https://github.com/mui-org/material-ui/issues/8183#issuecomment-343127262

> 1. That isn't the spec for contrast color it's the spec for the color pallette. The text is simply labels, so it doesn't matter that they're different.
> 2) our current color tables' labels don't match the spec in places if you compare them.

This is the best default for end users IMHO. If you _really_ want the docs color table labels closer to the labels in the spec (contrastThreshold = 3.1 doesn't exactly match either), we can override it in the docs.

Reply:
I won't argue for 0.1.